src/graphql/resolvers.ts

```diff
@@ -202,7 +202,7 @@ const resolvers = {
 
     children: async (parent: AreaType, _, { dataSources: { areas } }: Context) => {
       if (parent.children.length > 0) {
-        return await areas.findManyByIds(parent.children)
+        return (await areas.findManyByIds(parent.children)).sort(compareAreaLeftRightIndex)
```

Comment:
What do you think if we let Mongo handle the sorting here and in `findManyClimbsByUuids()`.  This way we don't have to write comparators. 
Something like:

```javascript

// inside AreaDataSource
 ...
 return await this.areaModel.find().
   where('metadata.leftRightIndex').
   in(parent.children).
   sort({ 'metadata.leftRightIndex': 1 })

```